common/models/user.js

```diff
@@ -475,13 +606,16 @@ module.exports = function(User) {
    */
 
   User.prototype.verify = function(options, fn) {
+    options = options || this.getVerifyOptions();
     fn = fn || utils.createPromiseCallback();
 
     var user = this;
     var userModel = this.constructor;
     var registry = userModel.registry;
     var pkName = userModel.definition.idName() || 'id';
-    assert(typeof options === 'object', 'options required when calling user.verify()');
+    assert(typeof options === 'object', 'verifyOptions are required when calling ' +
+      'user.verify(), please verify the implementation of your user model\'s ' +
+      'getVerifyOptions() method');
     assert(options.type, 'You must supply a verification type (options.type)');
```

Comment:
now that we default options with `getVerifyOptions()` the only case where options can be undefined is when user does not extend correctly this method